server/src/main/java/org/opensearch/index/MergeSchedulerConfig.java

```diff
@@ -130,19 +138,68 @@ public final class MergeSchedulerConfig {
         Property.NodeScope
     );
 
+    private final String indexName;
+    private IndexSettings indexSettings;
     private volatile boolean autoThrottle;
     private volatile int maxThreadCount;
     private volatile int maxMergeCount;
     private volatile double maxForceMergeMBPerSec;
+    private static volatile Boolean cachedAutoThrottleEnabledDefault;
```

Comment:
nit - rename this to clusterAutoThrottleEnabledDefault, clusterMaxThreadCountDefault & so on?